docker/scripts/cts/docker-entrypoint.sh

```diff
@@ -130,41 +130,41 @@ export ANT_OPTS="-Djava.endorsed.dirs=$CT_HOME/vi/glassfish7/glassfish/modules/e
 # TODO : Call tsJte 
 cd $TS_HOME/bin
 
-sed -i "s#report.dir=.*#report.dir=$CTS_HOME/ctsreport#g" ts.jte
-sed -i "s#work.dir=.*#work.dir=$CTS_HOME/ctswork#g" ts.jte
+sed -i.bak "s#report.dir=.*#report.dir=$CTS_HOME/ctsreport#g" ts.jte
```

Comment:
Please feel free to delete the folder docker/scripts as it is currently not used. It was added as a backup for the scripts in docker/*.sh and are all outdated now.